### DISCUSSION TITLE
Jump to Specific Buffer

### DISCUSSION THREAD

**Discussion by MickHarrigan** [helix-editor/helix#10936] [Ideas]
Currently the bufferline can only be navigated via `gn` or `gp`. I believe that the recent inclusion of `gw` for the jump mode would be a great inspiration for changing buffers.

For example pressing `gb` would make each of the buffer names into some string (again similarly to `gw`) that can then be chosen allowing switching to specific buffers without having to do anything involving numbered buffers or `gxn`/`gxb` where x is some amount of buffers to move.

**Comment by P4ll** [comment]
Seems to be solved via space + b
